projects/admin/src/app/routes/documents-route.ts

```diff
@@ -33,6 +35,17 @@ export class DocumentsRoute extends BaseRoute implements RouteInterface {
   /** Record type */
   readonly recordType = 'documents';
 
+  /**
+   * Constructor
+   * @param routeToolService - RouteToolService
+   */
+   constructor(
+    protected _routeToolService: RouteToolService,
+    protected _route: ActivatedRoute
```

Comment:
With the review of @vgranata PR yesterday, I figured out that `activatedRoute` is available from the `_routeToolService` :: `this._routeToolService.activatedRoute()`.
So you can remove this argument (and the entire constructor because it will remain the same as parent constructor) ; remove imports, ...